src/topp/IDFilter.cpp

```diff
@@ -180,485 +145,408 @@ class TOPPIDFilter :
     registerTOPPSubsection_("precursor", "Filtering by precursor RT or m/z");
     registerStringOption_("precursor:rt", "[min]:[max]", ":", "Retention time range to extract.", false);
     registerStringOption_("precursor:mz", "[min]:[max]", ":", "Mass-to-charge range to extract.", false);
-    registerFlag_("precursor:allow_missing", "When filtering by precursor RT or m/z, keep peptide IDs with missing precursor information ('RT'/'MZ' meta values)?");
```

Comment:
The functionality provided by this flag had been removed already (see https://github.com/OpenMS/OpenMS/blob/develop/src/topp/IDFilter.cpp#L302). I just cleaned up the left-over parameter.